packages/x-components/src/views/design-system/design-system-utilities.vue

```diff
@@ -474,6 +474,48 @@
         <div class="x-background--neutral-100 x-margin--left-04">Margin Left 04</div>
       </div>
     </article>
+
+    <article class="positions x-list x-list--wrap x-list--gap-06 x-list--align-start">
+      <h2 class="x-title2">Position</h2>
+      <div class="x-background--neutral-95 x-padding--03 x-absolute">Absolute</div>
+      <div class="x-background--neutral-95 x-padding--03 x-static">Static</div>
+      <div class="x-background--neutral-95 x-padding--03 x-fixed">Fixed</div>
+      <div class="x-background--neutral-95 x-padding--03 x-relative">Relative</div>
+      <div class="x-background--neutral-95 x-padding--03 x-sticky">Sticky</div>
+    </article>
+
+    <article class="positions x-list x-list--wrap x-list--gap-06 x-list--align-start">
+      <h2 class="x-title2">Line Clamp</h2>
+      <div
+        v-for="lines in 5"
+        :key="lines"
+        class="x-background--neutral-95"
+        :class="' x-line-clamp--' + ++lines"
+      >
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas auctor ligula et tristique
```

Comment:
Other utilities demos have titles or text with the variable values of the token they're representing. Should we do the same here? Like starting the paragraph with `Line clamp of {{ lines }} lines` and then continue with the lorem ipsum. It makes the demo more obvious.

Reply:
added title for that